dashboard/db/migrate/20240628202031_create_join_table_workshop_course_offering.rb

```diff
@@ -0,0 +1,9 @@
+class CreateJoinTableWorkshopCourseOffering < ActiveRecord::Migration[6.1]
+  def change
+    create_join_table :pd_workshops, :course_offerings, table_name: :pl_workshop_topics do |t|
```

Comment:
This is not something I'm going to be a stickler about, but I am curious why you didn't like the default `course_offerings_workshops` table name? Rails really favors convention, so I'm nervous when we change it.

I would support, when we add the Rails model associations, doing something like
```
has_and_belongs_to_many :topics, join_table: course_offerings_workshops
```
in pd_workshops.rb, if that would make things clearer!

Reply:
(that syntax will need to be tweaked ... will also need to include the `class_name` at the very least. [Doc](https://api.rubyonrails.org/v7.1.3.4/classes/ActiveRecord/Associations/ClassMethods.html#method-i-has_and_belongs_to_many))

Reply:
Oh I just wasn't sure if it wouldn't be clear or not. That sounds good to me, I'll update this to not rename!